src/inverseRobot.js

```diff
@@ -24,7 +24,24 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const robotValues = [];
+
+  for (const key in robot) {
+    if (robotValues.includes(robot[key])) {
+      return null;
+    }
+
+    robotValues.push(robot[key]);
+  }
+
+  const robotKeys = Object.keys(robot);
+  const invRobot = {};
+
+  for (let i = 0; i < robotValues.length; i++) {
+    invRobot[robotValues[i]] = robotKeys[i];
+  }
```

Comment:
Avoid connecting two separate arrays logically here.
If any of them changes between initialization and usage - your program can be broken.
Having keys you can access corresponding values.